viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -92,6 +92,7 @@
           return !_.isUndefined(selectionTarget.projectLinkData);
```

Comment:
Its not visible since it comes from viite-374 but in line 90 we have one "// TODO: 374 validate selection" in this selectedSingleClick control. What kind of validation should it be?